lib/logIn.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/material.dart';
+
+import 'package:todo/buttonItem.dart';
+import 'package:todo/inputItems.dart';
+
+class LogIn extends StatefulWidget {
+  const LogIn({Key? key}) : super(key: key);
+
+  @override
+  State<LogIn> createState() => _LogInState();
+}
+
+class _LogInState extends State<LogIn> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Todo Manager'),
+        centerTitle: true,
+        //backgroundColor: Colors.deepOrange,
+        elevation: 0,
+        flexibleSpace: Container(
+          decoration: const BoxDecoration(
+            gradient: LinearGradient(
+                begin: Alignment.center,
+                end: Alignment.bottomCenter,
+                colors: [Colors.deepOrangeAccent, Colors.deepOrangeAccent]),
+          ),
+        ),
+      ),
+      body: SingleChildScrollView(
+        child: Column(
+          children: [
+            Container(
+              height: MediaQuery.of(context).size.height * 0.2,
```

Comment:
Ukoliko imas ovako dupliranja u MediaQuery.of(context).size.height i .width mozes da deklarises u body Size = MediaQuery.of(context).size; i dole u kodu za height da stavis size.height * .2 i size.width * .1